nodemapper/src/gui/Builder/components/Flow.tsx

```diff
@@ -286,6 +324,20 @@ export const getNodeById = (id: string, nodes: Node[]): Node | null => {
   return null;
 };
 
+export const getNodeName = (node: Node): string => {
+  return node.data.config.name;
+};
+
+export const getNodeByName = (name: string, nodes: Node[]): Node | null => {
+  for (const node of nodes) {
+    console.log("Checking node: ", node.data.config.name, " for ", name);
```

Comment:
This could produce a lot of log messages, maybe this can be reduced to only log after finding or not finding at all?

Reply:
Yes, I don't think this line is needed. Removed.